lib/ansible/module_utils/basic.py

```diff
@@ -489,6 +489,9 @@ def heuristic_log_sanitize(data, no_log_values=None):
         output = remove_values(output, no_log_values)
     return output
 
+
+# TODO: mv/cp to ansible.utils? seems odd for plugin/filter/matchstuff.py to
+#       require module_utils for this...
```

Comment:
I'll clean up TODOs etc once  we cover most of the discussion.

(wish there was a good way to markup/annotate code inline with git without using comments...)